src/tabs/adjustments.html

```diff
@@ -2,7 +2,7 @@
     <div class="content_wrapper">
         <div class="tab_title" i18n="tabAdjustments"></div>
         <div class="cf_doc_version_bt">
-            <a id="button-documentation" href="https://github.com/betaflight/betaflight/wiki" target="_blank" rel="noopener noreferrer"></a>
+            <a id="button-documentation" href="" target="_blank" rel="noopener noreferrer"></a>
```

Comment:
Where go this buttons with the new code?

Reply:
Attributes are managed dynamically in gui.js

https://github.com/betaflight/betaflight-configurator/blob/master/src/js/gui.js#L303-L311